modules/web/web-vhost/src/main/java/com/enonic/xp/web/vhost/impl/VirtualHostServiceImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.enonic.xp.web.vhost.impl;
+
+import java.util.List;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import com.enonic.xp.web.vhost.VirtualHost;
+import com.enonic.xp.web.vhost.VirtualHostConfigService;
+import com.enonic.xp.web.vhost.impl.config.VirtualHostConfig;
+
+@Component(immediate = true)
+public class VirtualHostServiceImpl
+    implements VirtualHostConfigService
+{
+
+    private VirtualHostConfig virtualHostConfig;
+
+    @Activate
+    public VirtualHostServiceImpl( @Reference final VirtualHostConfig virtualHostConfig )
+    {
+        this.virtualHostConfig = virtualHostConfig;
+    }
+
+    @Override
+    public boolean isEnabled()
+    {
+        return virtualHostConfig.isEnabled();
+    }
+
+    @Override
+    public VirtualHost getVirtualHost( final String name )
```

Comment:
I now hardly see a need for such an API method.
1. In java one can easily use streams
2- in js one can use find